backend/apps/accounts/urls.py

```
@@ -31,14 +31,13 @@
     path("me/export/", ExportDataView.as_view(), name="me-export"),
     path("stats/", UserStatisticsView.as_view(), name="user-stats"),
     path("users/", UserListView.as_view(), name="user-list"),
-    # path("users/suggestions/", UserSuggestionsView.as_view(), name="user-suggestions"),
-    # path("profile/avatar/", AvatarUploadView.as_view(), name="avatar-upload"),
+    path("users/suggestions/", UserSuggestionsView.as_view(), name="user-suggestions"),
```

Ruff (0.15.21)

[error] Undefined name `UserSuggestionsView`

(F821)